composer.json

```diff
@@ -15,10 +15,12 @@
         "php": ">=5.3.2"
     },
     "require-dev": {
-        "react/event-loop": "0.3.*@dev"
+        "react/event-loop": "0.3.*@dev",
+        "react/promise": "~2.2"
     },
     "suggest": {
-        "react/event-loop": "For using event-loop based scheduling."
+        "react/event-loop": "For using event-loop based scheduling.",
+        "react/promise": "For converting promises to and from observables."
```

Comment:
Ok. I see now. I propose the following:
- move `react/promise` to suggest and `require-dev`
- Add the `toPromise` and `fromPromise` methods to it's own helper. Something like: `Rx\React\Promise::toPromise($observable)` and `Rx\React\Promise::fromPromise($promise)`? Perhaps the class name should be different to have no name clashes.

Down the line I'd like to move the react event loop scheduler there as well.